### ISSUE TITLE
Possible issue with version number: sbt warning that it cannot re-parse the jar 

### ISSUE THREAD

**Issue by hmf** [saddle/saddle#81]
Hello,

In #67 reported indicated that something was amiss with the version number.  More concretely the version is: `1.3.5-SNAPSHOT`, but the `version.sbt` shows `1.3.7-SNAPSHOT`.
When executing SBT commands I am getting: 

    [warn] Unable to reparse org.scala-saddle#saddle-core_2.12;1.3.5-SNAPSHOT from Sonatype OSS Snapshots, using Wed Jun 28 16:44:11 WEST 2017

A quick search led me to:

* https://stackoverflow.com/questions/37225775/idea-sbt-unable-to-reparse-warning
* https://github.com/sbt/sbt/issues/2650

Seeing as I do not have any local cache for Saddle, the version numbering may be the problem. 
Alternatively, this may be specific to snapshots (see second link). 

 Hope this information is of use.

**Comment by hntd187** [comment]
Yes, I realized that the snapshot version was wrong and I attempted to push a 1.3.6 version jar too, but it seems that it never got staged, hmm okay, I'll try and do this again. I'm sorry, but the versions are probably going to get mauled while I figure out how to do this properly.

**Comment by hmf** [comment]
No problem. Just though it might be useful when you release a version.
